closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -488,6 +542,11 @@ private static void setProperties(Properties props) {
                 propsOldValue = "";
             }
 
+            if (propsNewValue != null) {
+                // Add to list of properties set by the active profile.
+                alteredProperties.add(jdkPropsName);
+            }
```

Comment:
No need to remember the property name unless modifying it is to be disallowed.
```java
            if ((propsNewValue != null) && userEnabledFIPS && !allowSetProperties) {
                // Add to set of properties set by the active profile.
                unmodifiableProperties.add(jdkPropsName);
            }
```

Reply:
Updated.

Reply:
It should say "set" instead of "list" now.

Reply:
Changed the comment too.